src/store/modules/resetpassword/reducer/index.js

```diff
@@ -0,0 +1,31 @@
+import actionTypes from '../actionTypes';
+
+const {
+  PASSWORD_RESET_PENDING,
+  PASSWORD_RESET_SUCCESS,
+  PASSWORD_RESET_FAILED
+} = actionTypes;
+
+const initialState = {
+  isPending: false,
+  isSuccess: false,
+  error: null
+};
+
+const resetPassword = (state = initialState, action) => {
+  switch (action.type) {
+    case PASSWORD_RESET_PENDING:
+      return Object.assign({}, state, { isPending: true });
+    case PASSWORD_RESET_SUCCESS:
+      return Object.assign({}, state, { isPending: false, isSuccess: true });
+    case PASSWORD_RESET_FAILED:
+      return Object.assign({}, state, {
+        isPending: false,
+        error: action.error
+      });
+    default:
+      return state;
+  }
+};
```

Comment:
You may consider putting these `actions` in an array and then using the `includes` method to return them.

Reply:
Okay. thanks for the feedback